README.md

```diff
@@ -172,6 +172,21 @@ The default is 1 member to 1 group (deterministic run).
 :x: No implementation planned <br/>
 :soon: Work in progress
 
+# HofX case as a verification tool
+0. HofX case can be run separately as an independant verification tool.
+
+1. Instructions on how to run this case are as follows:
+
+   a. ./setup_case.sh -p HERA ../cases/hofx3d.yaml workflowtest_HOFX
+
+   b. In the workflow/cases/hofx3d.yaml file, da_settings: DA_ADT: True and the others false. 
+      Please add "/scratch2/NCEPDEV/marineda/Shastri.Paturi/Shastri.Paturi/rundir/workflowtest_QCexp_SST25" to SOCA_ANALYSIS. 
+      This has the 24 hr fcst run from Oct 1-31, 2011 with satellite SST assimilation.
+
+   c. The results of the run will be in RUNCDATE/Data.
+
+   d. A scatter plot between the obs_absolute_dynamic_topography@ObsValue and obs_absolute_dynamic_topography@hofx is one-way of looking at the results.
```

Comment:
I don't see any calls to plotting routines, how does the scatter plot is created?

Reply:
@flampouris, I will add a scatter plot routine to the JJOB_POST in the next version, as discussed.